ui/app/js/controllers/permissions_controller.js

```diff
@@ -76,50 +75,55 @@ function($scope, $routeParams, $location, $timeout, Permissions, Search, $modal,
   };
   /* End filtering */
 
-  $scope.openUpdateModal = function(user) {
-
-    var modalInstance = $modal.open({
-      templateUrl: 'permissions_modal.html',
-      controller: 'UserPermissionsCtrl',
-      backdrop: 'static',
-      // size: size,  // can be lg or sm
-      resolve: {
-        user: function () {
-          return user;
-        },
-        users: function () {
-          return $scope.users;
+    $scope.openUpdateModal = function(user) {
```

Comment:
Can you please revert the whitespace changes here? They've caused the indentation to be different than the rest of the file.

Reply:
Okay